components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml

```diff
@@ -115,7 +116,7 @@
     #end
   #end
   #if ("$!{pedigreeHtml}" != '')
-    (% id='pedigree' class='chapter clinical-info pedigree-family-section' %)(((
+    (% id='pedigree' class='clinical-info pedigree-family-section' %)(((
```

Comment:
This code is actually commented out and should have been removed.The change has no effect.